kolibri_instant_schools_plugin/assets/src/views/about/views/about/about.html

```diff
@@ -10,16 +10,15 @@
   <body>
 
     <img class="vodafone-logo" src="./assets/images/vodafone-logo.png">
-    <img class="instant-schools-logo" src="./assets/images/instant-schools-logo.png">
 
 
-    <!-- BEGIN EDITS-->
+    <!-- BEGIN CONTENT -->
```

Comment:
Make sure to make the same changes to faq.html for consistency.

Reply:
done